app/uk/gov/hmrc/fhregistrationfrontend/views/deregistration/deregistration_confirm.scala.html

```diff
@@ -14,137 +14,163 @@
-
-    @helper.CSRF.formField
-
-    @error_box(confirmationForm, ".deregistrationConfirm.")
-
-    @RadioGroup(
-      RadioGroupParams(
-        name = Some(confirmKey),
-        legend = Some(title),
-        legendAsHeading = true,
-        inline = true,
-        error = Some(Messages("fh.deregistrationConfirm.confirm." + getError(confirmationForm(confirmKey).error))),
-        hasErrors = confirmationForm(confirmKey).hasErrors,
-        value = confirmationForm(confirmKey).value,
-        radios = List(
-          RadioParams(
-            "true",
-            Messages("fh.generic.yes"),
-            Some("emailContainer")
-          ),
-          RadioParams(
-            "false",
-            Messages("fh.generic.no")
-          )
-        )
-      )
-    )
-
-    <div class="js-hidden panel" id="emailContainer">
-
-    @if(defaultEmail.filterNot(_.isEmpty) == None) {
-
-      <input type="hidden" name="@usingDefaultEmailKey" value="false"/>
-      <legend class="form-label-bold">@Messages("fh.confirmationEmail.noDefaultEmail.legend")</legend>
-      @EmailConfirmation(
-        EmailConfirmationParams(
-          name = alternativeEmailKey,
-          label = Messages("fh.confirmationEmail.noDefaultEmail.email.label"),
-          labelConfirmation = Messages("fh.confirmationEmail.noDefaultEmail.emailConfirm.label"),
-          error = Messages("fh.confirmationEmail.alternativeEmail.email.error.required"),
-          isEmail = true,
-          hasErrors =  confirmationForm(s"$alternativeEmailKey.email").hasErrors || confirmationForm(s"$alternativeEmailKey.emailConfirmation").hasErrors,
-          value = confirmationForm(s"$alternativeEmailKey.email").value,
-          valueConfirmation = confirmationForm(s"$alternativeEmailKey.emailConfirmation").value
-        )
-      )
-
-    } else {
-      <input type="hidden" name="@defaultEmailKey" value="@{defaultEmail.get}"/>
-      <legend class="form-label-bold">@Messages("fh.confirmationEmail.hasDefaultEmail.legend", defaultEmail.get)</legend>
-      @RadioGroup(
-        RadioGroupParams(
-          name = Some(usingDefaultEmailKey),
-          legend = Some(Messages("fh.confirmationEmail.use_default_email.confirm.label")),
-          hasErrors = confirmationForm(usingDefaultEmailKey).hasErrors,
-          error = Some(Messages("fh.confirmationEmail.use_default_email.confirm.error")),
-          value = confirmationForm(usingDefaultEmailKey).value,
-          radios = List(
-            RadioParams(
-              "true",
-              Messages("fh.generic.yes")
+@emailConditionalHtml =   {
+    <div class="govuk-body">
+        @viewHelpers.govukInput(Input(
+            id = s"$alternativeEmailKey.email",
+            name = s"$alternativeEmailKey.email",
+            inputType = "email",
+            label = Label(
+                isPageHeading = true,
+                content = HtmlContent(s"<strong>${messages("fh.confirmationEmail.alternative_email.error.required")}</strong>")
```

Comment:
instead of strong, you can write this as
`label = Label(
        isPageHeading = true,
        content = Text(messages("fh.confirmationEmail.alternative_email.error.required")),
        classes = "govuk-label--s",
      ),`

Reply:
Done